test/runtests.jl

```diff
@@ -531,6 +529,31 @@ end
         end
     end
 
+    @testset "KNN" begin
```

Comment:
Don't we also want a test case where we can demonstrate that knn is sufficiently better than mean imputation? For example, we had a test where we expected svd imputation to be at least twice as accurate as mean imputation.

Reply:
After adding tests, I found I made huge mistake. I need to rewrite code. Thank you for pointing that.

Reply:
No worries, sorry I didn't notice that sooner :)